package.json

```diff
@@ -23,7 +24,7 @@
     "contracts:redeploy:devnet": "meta exec \"npx truffle migrate --network rskdevnet --reset\" --include-only $npm_package_config_contracts",
     "contracts:deploy:testnet": "meta exec \"npx truffle migrate --network rsktestnet\" --include-only $npm_package_config_contracts",
     "contracts:deploy:mainnet": "meta exec \"npx truffle migrate --network rskmainnet\" --include-only $npm_package_config_contracts",
-    "contracts:create-config": "node scripts/createConfig.js rns storage notifier",
+    "contracts:create-config": "node scripts/createConfig.js rns storage notifier rif-notifier",
```

Comment:
rif-notifer can be removed

Reply:
i wanted to separate rif-marketplace-notifier from rif-notifier. Anyways, no issues we can use the same parameter